src/ert/run_models/base_run_model.py

```diff
@@ -395,7 +392,11 @@ def send_end_event(self) -> None:
         self.send_event(
             EndEvent(
                 failed=self._failed,
-                failed_msg=self.getFailMessage(),
+                msg=(
+                    self.getFailMessage()
+                    if self._failed
+                    else self._current_iteration_label
```

Comment:
Looks like this is always `Experiment completed.`, can hard code that here and remove it from the run models.